rumqttc/src/lib.rs

```diff
@@ -902,3 +901,313 @@ mod test {
         let _mqtt_opts = MqttOptions::new("", "127.0.0.1", 1883).set_clean_session(true);
     }
 }
+
+// Dummy broker used in unit tests
+#[allow(dead_code)]
```

Comment:
needed to add this because we only use this in unit tests in `eventloop` module. We might also use this in other modules so added it here instead of in unit tests module directly.